apps/prime/base.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import time
+
+from locust import TaskSequence
+from locust import task
+from locust import events
+
+from bravado.swagger_model import load_file
+from bravado.client import SwaggerClient
+from bravado.requests_client import RequestsClient
+
+from bravado_core.formatter import SwaggerFormat
+from bravado_core.exception import SwaggerMappingError
+from bravado.exception import HTTPError
+
+
+def get_swagger_config():
```

Comment:
You shouldn't need top copy this here. I'd suggest creating a `common/` module to share this code in.